src/pagination/pagination-nav/pagination-nav.component.ts

```diff
@@ -141,6 +143,22 @@ export class PaginationNav {
 		this.previousItemText.override(valueWithDefaults.PREVIOUS);
 	}
 
+	/**
+	 * Sets the pagination nav size
+	 */
+	@Input() size: "sm" | "md" | "lg" = "lg";
+
+	// Size
+	@HostBinding("class.cds--layout--size-sm") get sizeSm() {
+		return this.size === "sm";
+	}
+	@HostBinding("class.cds--layout--size-md") get sizeMd() {
+		return this.size === "md";
+	}
+	@HostBinding("class.cds--layout--size-lg") get sizeLg() {
+		return this.size === "lg";
+	}
```

Comment:
```suggestion
	@HostBinding("class.cds--layout--size-sm") get smallSize() {
		return this.size === "sm";
	}
	@HostBinding("class.cds--layout--size-md") get mediumSize() {
		return this.size === "md";
	}
	@HostBinding("class.cds--layout--size-lg") get largeSize() {
		return this.size === "lg";
	}
```

Just updating the names to match the layout names in button 🤔